CMakeLists.txt

```diff
@@ -34,5 +36,6 @@ add_subdirectory(tests)
 
 set(DOXYGEN_MAINPAGE_MD README.md)
 set(DOXYGEN_EXTRA_INPUT ${PROJECT_SOURCE_DIR}/README.md)
-include(CPackConfig)
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
+include(CommonCPack)
```

Comment:
just a minor detail, but in other projects I grouped the CommonCPack lines above the doxygen lines:

    set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
    include(CommonCPack)

    set(DOXYGEN_MAINPAGE_MD README.md)
    set(DOXYGEN_EXTRA_INPUT ${PROJECT_SOURCE_DIR}/README.md)
    include(DoxygenRule)